core/src/main/java/org/ehcache/EhcacheManager.java

```diff
@@ -105,6 +107,7 @@ public EhcacheManager(Configuration config, ServiceLocator serviceLocator, boole
   @Override
   public void removeCache(final String alias) {
     statusTransitioner.checkAvailable();
+    MDC.put("alias", alias);
```

Comment:
ehcache might not be the only user of the MDC, and "alias" is quite a generic term. I think this should be prefixed to avoid potential name clashes.